app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx

```diff
@@ -28,7 +28,7 @@ import { activeProjectSelector } from 'controllers/user';
 import { showModalAction } from 'controllers/modal';
 import { SpinningPreloader } from 'components/preloaders/spinningPreloader';
 import { DASHBOARD_PAGE_EVENTS } from 'components/main/analytics/events';
-import { CHARTS, MULTI_LEVEL_WIDGETS_MAP, NoDataAvailable } from 'components/widgets';
+import { CHARTS, MULTI_LEVEL_WIDGETS_MAP, NoDataAvailable, ErrorMessage } from 'components/widgets';
```

Comment:
I think will be better to import it directly from `components/main/errorMessage`

Reply:
Oh, sure. Something weird has been left after moving components around